wadas/ai/pipeline.py

```diff
@@ -52,12 +52,16 @@ def __init__(
 
         # Initializing the MegaDetectorV5 model for image detection
         logger.info("Initializing detection model to device %s...", self.detection_device)
-        if megadetector_version == "v5":
-            detection_csl = OVMegaDetectorV5
-        elif megadetector_version == "v6":
-            detection_csl = OVMegaDetectorV6
-        else:
-            raise ValueError("Invalid MegaDetector version: " + megadetector_version)
+        match megadetector_version:
+            case "MDV5-yolov5":
+                detection_csl = OVMegaDetectorV5
+            case "MDV6b-yolov9c":
+                detection_csl = OVMegaDetectorV6
+            case "MDV6-yolov10n":
+                detection_csl = OVMegaDetectorV6
+            case _:
+                raise ValueError("Invalid MegaDetector version: " + megadetector_version)
+
```

Comment:
A simple map can be used in this case:

NAME_TO_DETECTOR = {
    "MDV5-yolov5": OVMegaDetectorV5,
    "MDV6b-yolov9c": OVMegaDetectorV6,
    "MDV6-yolov10n": OVMegaDetectorV6,
}

and in the above code:
    if not (detection_csl := NAME_TO_DETECTOR.get(megadetector_version)):
        raise ValueError("Invalid MegaDetector version: " + megadetector_version)